magicctapipe/conftest.py

```diff
@@ -50,6 +51,7 @@
 @pytest.fixture(scope="session")
 def temp_DL1_gamma(tmp_path_factory):
     return tmp_path_factory.mktemp("DL1_gammas")
+        
```

Comment:
no need for 3 lines

Reply:
Do you mean blank lines or the folder?

Reply:
I meant the blank lines.  But I don't know if we have a black setting somewhere which uniformise behaviour for this. I personally use the convention (from my IDE and things I have seen in projects) : 2 blank  lines between classes and functions in a module. One blank line between elements in a class. Never more than 2 blank lines in general.

Reply:
I removed the blank lines. I generally run black on everything and hope its setting will create a nice output. BTW, I will check the code layout when we finish the greatest changes